rmf_demos_fleet_adapter/README.md

```diff
@@ -0,0 +1,171 @@
+# rmf_demos_fleet_adapter
+
+This is an implementation of the python based [fleet adapter template](https://github.com/open-rmf/fleet_adapter_template) on selected RMF demo worlds: Hotel, Office, Airport Terminal and Clinic. 
+
+The fleet adapter uses FastAPI to:
```

Comment:
Got it, updated the readme. I hope the brief explanation is sufficient, didn't elaborate too much on the endpoints since that is done in the following sections